pagerduty/resource_pagerduty_incident_workflow_test.go

```diff
@@ -219,46 +441,152 @@ func testAccCheckPagerDutyIncidentWorkflowExists(n string) resource.TestCheckFun
 
 func TestFlattenIncidentWorkflowStepsOneGenerated(t *testing.T) {
 
-	n := &pagerduty.IncidentWorkflow{
+	iw := &pagerduty.IncidentWorkflow{
 		Steps: []*pagerduty.IncidentWorkflowStep{
 			{
 				ID:   "abc-123",
-				Name: "something",
+				Name: "step1",
 				Configuration: &pagerduty.IncidentWorkflowActionConfiguration{
+					ActionID: "step1-action-id",
 					Inputs: []*pagerduty.IncidentWorkflowActionInput{
 						{
-							Name:  "test1-value",
+							Name:  "step1-input1",
 							Value: "test1-value",
 						},
 						{
-							Name:  "test2-value",
+							Name:  "step1-input2",
 							Value: "test2-value",
 						},
 						{
-							Name:  "test3-value",
+							Name:  "step1-input3",
 							Value: "test3-value",
 						},
 					},
+					InlineStepsInputs: []*pagerduty.IncidentWorkflowActionInlineStepsInput{
+						{
+							Name: "step1-inlineinput1",
+							Value: &pagerduty.IncidentWorkflowActionInlineStepsInputValue{
+								Steps: []*pagerduty.IncidentWorkflowActionInlineStep{
+									{
+										Name: "step1a",
+										Configuration: &pagerduty.IncidentWorkflowActionConfiguration{
+											ActionID: "step1a-action-id",
+											Inputs: []*pagerduty.IncidentWorkflowActionInput{
+												{
+													Name:  "step1a-input1",
+													Value: "inlineval1",
+												},
+												{
+													Name:  "step1a-input2",
+													Value: "inlineval2",
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+						{
+							Name: "step1-inlineinput2",
+							Value: &pagerduty.IncidentWorkflowActionInlineStepsInputValue{
+								Steps: []*pagerduty.IncidentWorkflowActionInlineStep{
+									{
+										Name: "step1b",
+										Configuration: &pagerduty.IncidentWorkflowActionConfiguration{
+											ActionID: "step1b-action-id",
+											Inputs: []*pagerduty.IncidentWorkflowActionInput{
+												{
+													Name:  "step1b-input1",
+													Value: "inlineval3",
+												},
+												{
+													Name:  "step1b-input2",
+													Value: "inlineval4",
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
 				},
 			},
 		},
 	}
-	o := map[string][]string{
-		"abc-123": {"test1-value", "test2-value"},
+	specifiedSteps := []*SpecifiedStep{
+		&SpecifiedStep{
```

Comment:
Could please remove the redundant types declarations on slice elements like `&SpecifiedStep` and `[]*SpecifiedStep` for map values?

Reply:
This is my first time using golang- could you clarify this a bit for me? Is there some automatic magic that happens with slice items that convert structs to a pointer for you, or something like that? So this becomes `specifiedSteps := []SpecifiedStep{ SpecifiedStep{ ... }, ... }` ?